ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -952,6 +954,7 @@ private Long incrementalDump(Path dumpRoot, DumpMetaData dmd, Path cmRoot, Hive
           }
         }
         for(String matchedDbName : Utils.matchesDb(hiveDb, work.dbNameOrPattern)) {
+          ExportService exportService = new ExportService(conf);
```

Comment:
Please move the ExportService object creation higher than a database level.

Reply:
Done